Resources/doc/mapping.rst

```diff
@@ -0,0 +1,207 @@
+Mapping
+=======
+
+Elasticsearch bundle requires mapping definition in order for it to work with index.
+
+Mapping configuration
+---------------------
+
+Here’s an example of configuration with tokenizer’s, filters and analyzer’s definitions:
```

Comment:
tokenizers', filters' and analyzers' definitions. Although that sounds not that good - maybe `Here's an example of configuration containing the definitions of tokenizer, filter and analyzer` would be better?